composer.json

```diff
@@ -22,15 +22,14 @@
         ]
     },
     "require": {
-        "symfony/framework-bundle": "2.*",
-        "symfony/console": "2.*",
+        "symfony/framework-bundle": "^2.7|^3.0",
+        "symfony/console": "^2.7|^3.0",
         "maxmind-db/reader": "0.3.*"
     },
     "require-dev": {
         "phpunit/phpunit": "*",
         "composer/composer": "*@alpha"
     },
-    "minimum-stability": "stable",
```

Comment:
Basically because that's not relevant for bundles, and should only be included in your own project. It is a root-only field, as you can see here: https://getcomposer.org/doc/04-schema.md#minimum-stability